package-ucan.json

```diff
@@ -2,19 +2,28 @@
   "name": "@fireproof/ucan",
   "version": "0.18.0",
   "description": "",
-  "main": "./dist/browser/index.cjs",
-  "module": "./dist/browser/index.esm.js",
+  "main": "./index.cjs",
+  "module": "./index.js",
   "exports": {
     ".": {
-      "import": "./dist/browser/index.esm.js",
-      "require": "./dist/browser/index.cjs",
-      "types": "./dist/types/index.d.ts",
-      "script": "./dist/browser/index.iife.js"
+      "import": "./index.js",
+      "require": "./index.cjs",
+      "script": "./index.global.js",
+      "types": "./index.d.ts"
     }
   },
-  "browser": "./dist/index.browser.iife.js",
-  "types": "./dist/types/index.d.ts",
-  "files": ["dist/node", "dist/browser", "dist/types", "src/server.ts"],
+  "browser": "./index.global.js",
+  "types": "./index.d.ts",
+  "files": [
+    "index.js",
+    "index.cjs",
+    "index.d.ts",
+    "index.d.cts",
+    "index.js.map",
+    "index.cjs.map",
+    "chunk-*.js",
+    "browser-*.js"
+  ],
```

Comment:
Missing couple of things I'm assuming?
```
index.global.js
*.map // Or chunk-*.map + browser-*.map + index.global.js.map
README.md
LICENSE.md
```

Reply:
I've just pushed one more commit to try and address these.  The LICENSE.md is getting pulled in anyway, even though it isn't in that list.  And I updated the package.json script to copy the README from src folder, in the same way that we do for the other packages.  It isn't included in the tgz, but I believe it will use the correct README when published to the website.  If we want to include README in all the tgz, we should update all the gateways to be consistent.